.github/workflows/_link_check.yml

```diff
@@ -23,15 +21,16 @@ jobs:
       ref: ${{ inputs.ref }}
       script: |
         ./scripts/lint_urls.sh $(
-          [ "${{ github.event_name }}" = "pull_request" ] \
-            && git diff --name-only ${{ github.event.pull_request.base.sha }}...${{ github.event.pull_request.head.sha }} \
-          || [ "${{ github.event_name }}" = "push" ] \
-            && git diff --name-only ${{ github.event.before }}...${{ github.sha }}
+          { [ "${{ github.event_name }}" = "pull_request" ] \
```

Comment:
Looking at this line, should this be `==` instead? Review bash script is tricky :)

Reply:
Ah, never mind, this looks right https://stackoverflow.com/questions/2237080/how-to-compare-strings-in-bash